### DISCUSSION TITLE
Is the password supposed to be visible in the form data?

### DISCUSSION THREAD

**Discussion by wira-putra** [aminalaee/sqladmin#427] [General]
Hi everyone,
I have tried authentication example in https://aminalaee.dev/sqladmin/authentication/, but when I use 'inspect' in the browser at `/admin/login`, the form data (including the password) is visible.

<img width="505" alt="image" src="https://user-images.githubusercontent.com/62983838/217528903-bedde320-b378-48f9-96c9-9317152068f2.png">

Is it normal or a bug? If so, maybe we should note it in the docs for security concern.

Thank you in advance

**Comment by aminalaee** [comment]
Hey,

Thanks. Yes I think it's very normal and expected to send plain text password in POST body as most other tools do.
But I think it's worth mentioning that HTTPS is recommended to be used with authentication.

Feel free to update the docs with a PR.
